source/codegen/metadata/nifake/enums.py

```diff
@@ -117,37 +45,22 @@
         'generate-mappings': True,
         'values': [
             {
-                'documentation': {
```

Comment:
Note: Removed unnecessary documentation from this file while I was here.

Reply:
unnecessary! how else would we know the TMNT weapons of choice